packages/aws-cdk/test/cdk-toolkit.test.ts

```diff
@@ -375,7 +382,7 @@ describe('watch', () => {
           fakeChokidarWatcherOn.fileEventCallback('unlink', 'my-file3'),
```

Comment:
We should probably add one more event here (the whole point of this test is that there are more events than calls to `cdk deploy`, and right now this is not true).

Reply:
sounds good